Update sidenav order for JS (#11008)
getsentry/sentry-docs: docs/platforms/javascript/common/best-practices/index.mdx

---

> title: Niche Use Cases

Reviewer: Not sure if niche use cases is the best title here (especially for non-native speakers).

Some alternatives:

- Special Use Cases
- Special Case Scenarios
- Advanced Use Cases

Author: I like Special Use Cases!

> title: Niche Use Cases

Author: ```suggestion
title: Special Use Cases
```

> description: "Learn how to set up Sentry for specific niche scenarios with these best practice guides."

Author: ```suggestion
description: "Learn how to set up Sentry for several specific use cases with these best practice guides."
```

> description: "Learn how to set up Sentry for these specific niche scenarios with these best practice guides."

Reviewer 2: ```suggestion
description: "Learn how to set up Sentry for specific niche scenarios with these best practice guides."
```
